WeakAuras/GenericTrigger.lua

```diff
@@ -2130,6 +2130,37 @@ do
   end
 end
 
+local watchUnitChange
```

Comment:
You mentioned that there is a possible refactoring opportunity in your commit message. What did you have in mind?

Reply:
Currently, triggers subscribe to events. But for some triggers, the events fire a lot more often than the triggers actually care about. E.g. SPELL_COOLDOWN_CHANGED is fired whenever a single spell cd changes, but triggers for all spell watch the same signal, and thus evaluate.

That's clearly not needed.

This sidesteps that issue at the cost of creating lots of fake events, but the problem is much more general.

Reply:
Ok, thanks